proto/table_unmarshal.go

```diff
@@ -1514,6 +1515,9 @@ func unmarshalStringValue(b []byte, f pointer, w int) ([]byte, error) {
 		return nil, io.ErrUnexpectedEOF
 	}
 	v := string(b[:x])
+	if !utf8.ValidString(v) {
```

Comment:
Just a thought, but we could test the values with `utf8.Valid` before converting them to a string.

I mean, the code is essentially the same, but if we can validate it sooner, why not?

Reply:
There's no performance penalty for `utf8.ValidString` since it doesn't convert the string to a `[]byte` and call `utf8.Valid` under the hood.

I'm going to keep it as is since `utf8.ValidString(v)` is more readable than `utf8.Valid(b[:x])`, where you don't have to consciously think about slicing `b`.

Reply:
Indeed, good call. 👍